services/optimiser/testing/test_shift.py

```diff
@@ -0,0 +1,44 @@
+import pytest as pytest
+
+from domain.base import Session, Engine
+from services.optimiser.input_processing import *
+
+# Todo: Mock session
+
+@pytest.fixture
+def session():
+    new_session = Session(bind=Engine)
+    yield new_session
+
+    new_session.close()
+
+
+@pytest.fixture
+def request_id():
+    return '356'
+
+
+def test_get_input_qualrequirements(session, request_id):
+    print(get_input_qualrequirements(session, request_id))
```

Comment:
Since we're not really using pytest yet and because we haven't yet got a way of testing against a mock (simpler) database, I don't mind too much that these aren't really test cases. However, could you please add a TODO somewhere in this file to write actual test cases for each of the test functions. These should have an input, expected output and actual output,